Libraries/include/MathGeoLib/Geometry/Frustum.cpp

```diff
@@ -203,7 +203,7 @@ float3 Frustum::FarPlanePos(float x, float y) const
 	else
 	{
 		float3 right = WorldRight();
-		return pos + front * farPlaneDistance
+		return pos + front * farPlaneDistance/2
```

Comment:
Why this change?

Reply:
Orthographic frustums, when rendered by dd::frustum, render its far plane at the double specified far plane distance, so, trying to fix it, I modified, this, but its already solved. So back to normal.